src/App.js

```diff
@@ -6,7 +6,27 @@ import moviesFromServer from './api/movies.json';
 export class App extends Component {
   state = {};
```

Comment:
задавай значения по-умолчанию
```suggestion
  state = {
    query: '',
  };
```